### ISSUE TITLE
Automatic email outputs that cronjob cannot touch linked files.

### ISSUE THREAD

**Issue by lukeconibear** [wrfchem-leeds/WRFotron#19]
**What happened**:
Automatic email outputs that cronjob cannot touch linked files.

```bash
touch: cannot touch ‘./amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00/scheme_input_emission_data.ncl’: Permission denied
touch: cannot touch ‘./amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00/preprocess_emissions_routines.ncl’: Permission denied
touch: cannot touch ‘./amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00/README.exo_coldens’: Permission denied
touch: cannot touch ‘./amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00/moz0002.nc’: Permission denied
touch: cannot touch ‘./amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00/sum_sector_emiss_wrfchemi.py’: Permission denied
```

**What you expected to happen**:
The cronjob to touch all files, including linked files.

**Environment**:
Compilation = CEMAC.

**Comment by lukeconibear** [comment]
The cronjob runs the `~/.not_expire.sh` script every month at `****-**-04 04:00:00`, which contains:
```bash
cd /nobackup/${USER}
find . -exec touch -ah {} \;
find . -exec touch -a {} \;
```

Runs with the CEMAC compilation create links to files on `/nobackup/WRFChem`. 

The links are updated correctly via the first find command (`find . -exec touch -ah {} \;`) as proven by the output below:
```bash
[earlacoa@login2.arc4 2010-06-11_18:00:00-2010-06-13_00:00:00]$ cd /nobackup/earjba/amazon_simulation_WRFChem4.2_test/run/base/2010-06-11_18:00:00-2010-06-13_00:00:00
[earlacoa@login2.arc4 2010-06-11_18:00:00-2010-06-13_00:00:00]$ stat emission_script_data_QH.ncl
  File: 'emission_script_data_QH.ncl' -> '/nobackup/WRFChem/WRF_UoM_EMIT/emission_script_data_QH.ncl'
  Size: 58              Blocks: 0          IO Block: 4096   symbolic link
Device: 9b7c8ceh/163039438d     Inode: 144116322870061857  Links: 1
Access: (0777/lrwxrwxrwx)  Uid: (446082/  earjba)   Gid: ( 7400/     geo)
**Access: 2020-11-04 04:00:04.000000000 +0000**
Modify: 2020-11-01 12:10:40.000000000 +0000
Change: 2020-11-04 04:00:04.000000000 +0000
 Birth: -
```

The cause of the email is that the second find command (`find . -exec touch -a {} \;`) for touching non-links, also tries to touch the files on `/nobackup/WRFChem`, which are restricted. But this isn't a problem as they are already updated.

In summary, the files are correctly updated and this automatic email can be ignored.

**Comment by lukeconibear** [comment]
The `find . -exec touch -a {} \;` line is superfluous and can be removed.
